### DISCUSSION TITLE
Set snatched to Failed after X hours

### DISCUSSION THREAD

**Discussion by sufd4** [pymedusa/Medusa#5226] [Ideas & Feature requests]
Sometimes with certain failures it's not reported back to Medusa correctly and an episode is forever stuck as 'Snatched'. Can you have a setting to allow it to automatically change that to failed and search again if it doesn't complete after a certain number of hours?

Similar to this option on SickRageTV 'the feature/option can be found under search clients called '**Enable failed snatch handling** '

Thanks

**Comment by a10kiloham** [comment]
I'd find this useful

**Comment by p0psicles** [comment]
The thing with this option is that a time based failing, does not work well with torrents.

So that would leave it mostly usefull for nzb users. But, nzb users are using sabnzbd or nzbget. And both have an api which can be used to actively check on the status of downloads. A while ago we discussed this. And back then at least the team voted to implement this active download checking. Sonarr also uses this.

But then it still comes down to people willing and having the time to implement it.

**Comment by sufd4** [comment]
Thanks. Is there any timeline for implementing active checking in sab? (or where it falls within your priorities)

**Comment by p0psicles** [comment]
With the currently activity. Maybe early 2019. Or we get some new contributors.
